src/store/reducers/sign_in.js

```diff
@@ -1,7 +1,7 @@
 import * as actionTypes from '../types';
 
 const initialState = {  
-  formIsValid: false,
+  formIsValid: true,
```

Comment:
What the reason here? seems logic is broken, form is invalid by default (empty)

Reply:
done. Should be false for both (default).